### ISSUE TITLE
Autosave Neon manifest using PouchDB

### ISSUE THREAD

**Issue by JRegimbal** [DDMAL/Neon#395]
Use PouchDB to save the manifest Neon is working on after editing actions. If the user loads a manifest file but the PouchDB manifest is newer (as specified by the timestamp), ask the user if they would prefer to load the autosaved version.

This restores autosave functionality from previous versions.

**Comment by JRegimbal** [comment]
Waiting on manifest features to be added to develop so that this can be a separate pull request.
